index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mobily First</title>
+  <!-- 
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap"
+    rel="stylesheet"
+    />
+
+  -->
+    <link rel="stylesheet" href="style.css">
+  
+    
+
+      <script
+        src="https://kit.fontawesome.com/26c38390b4.js"
+        crossorigin="anonymous"
+        ></script>
+  
+  </head>
+  <body>
+    
+      <nav class="navbar">
+        <p class="brand-title">My logo</p>
+        <i class="fas fa-bars"></i>
```

Comment:
- [ ] Your logo looks good 👍  , but for  semantic reason, kindly use an `a` tag 👍